plugins/modules/ome_application_network_proxy.py

```diff
@@ -173,56 +252,102 @@ def validate_check_mode_for_network_proxy(payload_diff, module):
 
 
 def get_payload(module):
+    """
+    Generates a payload for the network proxy configuration.
+
+    Args:
+        module (object): The Ansible module object.
+
+    Returns:
+        dict: The payload for the network proxy configuration.
+    """
     params = module.params
     proxy_payload_map = {
         "ip_address": "IpAddress",
         "proxy_port": "PortNumber",
         "enable_proxy": "EnableProxy",
         "proxy_username": "Username",
         "proxy_password": "Password",
-        "enable_authentication": "EnableAuthentication"
+        "enable_authentication": "EnableAuthentication",
+        "ignore_certificate_validation": "SslCheckDisabled",
+        "proxy_exclusion_list": "ProxyExclusionList"
     }
     backup_params = params.copy()
-    remove_keys = ["hostname", "username", "password", "port", "ca_path", "validate_certs", "timeout"]
+    remove_keys = ["hostname", "username", "password", "port", "ca_path", "validate_certs",
+                   "timeout", "update_password"]
     remove_unwanted_keys(remove_keys, backup_params)
```

Comment:
what is the use of it, just want to understand, as we are capturing the output of this functions, if we pass one arguments backup_params, it process that one but not returning any value.

Reply:
okay, understood, here, the argument uses the same memory as the variable. so it's removing the keys.